docs/cloud-adoption/governance/governance-single-team.md

```diff
@@ -86,7 +84,19 @@ The lowest level of management scope is at the **resource** level. Operations ap
 ![The **workload owner** creates a resource](../_images/governance-1-8.png)
 *Figure 8. The workload owner creates a resource and inherits the built-in owner role at the resource scope.*
 
-The **workload owner** inherits the owner role at the resource scope, which means the workload owner has all permissions for the virtual network. 
+The **workload owner** inherits the owner role at the resource scope, which means the workload owner has all permissions for the virtual network.
+
+## Implementing the basic resource access management model
+
+Let's move on to learn how to implement the governance model designed earlier. 
+
+To begin, your organization requires an Azure account. If your organization has an existing [Microsoft Enterprise Agreement](https://www.microsoft.com/licensing/licensing-programs/enterprise.aspx) that does not include Azure, Azure can be added by making an upfront monetary commitment. See [licensing Azure for the enterprise](https://azure.microsoft.com/pricing/enterprise-agreement/) for more information. 
+
+When your Azure account is created, you specify a person in your organization to be the Azure **account owner**. An Azure Active Directory (Azure AD) tenant is then created by default. Your Azure **account owner** must [create the user account](/azure/active-directory/add-users-azure-active-directory) for the person in your organization who is the **workload owner**. 
+
+Next, your Azure **account owner** must [create a subscription](https://docs.microsoft.com/partner-center/create-a-new-subscription) and [associate the Azure AD tenant](/azure/active-directory/fundamentals/active-directory-how-subscriptions-associated-directory) with it.
+
+Finally, now that the subscription is created and your Azure AD tenant is associated with it, you can [add the **workload owner** to the subscription with the built-in **owner** role](/azure/billing/billing-add-change-azure-subscription-administrator#add-an-rbac-owner-for-a-subscription-in-azure-portal).
 
 ## Summary
```

Comment:
Maybe add a sentence in the summary to the effect of: This is a simple governance model, with only one workload owner, but it's the basis for governing more complex teams/workloads?

Reply:
Unfortunately it's really not though :/ 
I see what's missing though - we used to have "deploy a basic workload" as a next step, so I added that back in.